modules/Bio/EnsEMBL/G2P/Utils/Downloads.pm

```diff
@@ -24,12 +25,26 @@ use base qw(Exporter);
 our @EXPORT_OK = qw( download_data );
 
 my $gfd_attributes = {};
-my $gfd_create_dates = {};
+my $gfd_panel_create_dates = {};
 my $gfid2synonyms = {};
 my $confidence_category_attribs = {};
 my $allelic_requirement_attribs = {};
 my $mutation_consequence_attribs = {};
 
+=head2 download_data
+  Arg [1]    : String $downloads_dir - download file to this directory
+  Arg [2]    : String $file_name - name of download file
+  Arg [3]    : String $registry_file - for connecting to the G2P database
+  Arg [4]    : Boolean $is_logged_in - stores if the user is logged in
+  Arg [5]    : Arrayref $user_panels - all panels that a user can edit
```

Comment:
Now the $user_panels is not being used in the code.

Reply:
You are right. I will remove the $user_panels and also update the gene2phenotype_app code.